### PR TITLE
Modifies the prompt for the cloud name to include cloud type

### PR DESCRIPTION
**Pull request by kat-co**
Note that this is based upon this [PR](https://github.com/juju/juju/pull/6598). Look at the latest commit for the (small) diff.

### CODE REVIEW

File: cmd/juju/cloud/add.go [hunk 1]
```diff
@@ -182,7 +186,7 @@ func queryName(cloudMetadataStore CloudMetadataStore, pollster *interact.Pollste
 	}
 
 	for {
-		name, err := pollster.Enter("a name for the cloud")
+		name, err := pollster.Enter(fmt.Sprintf("a name for your %s", cloudType))
```
Comment (bz2) [inline cmd/juju/cloud/add.go]: Is this clearer as "a name for your %s cloud". It's a little funny for 'maas', but makes more sense for 'manual' and is fine with 'openstack', 'vsphere', and so on.

Comment (kat-co) [inline cmd/juju/cloud/add.go]: Good idea, implemented.

**Comment by kat-co** [comment]
$$merge$$

**Comment by kat-co** [comment]
Spurious error.

$$merge$$

**Comment by kat-co** [comment]
Spurious error.

$$merge$$

**Pull request merged**